index.js

```diff
@@ -94,10 +99,12 @@ function send(ctx, path, opts) {
       throw err;
     }
 
+    if (setHeaders) setHeaders(ctx.res, path, stats);
```

Comment:
Maybe someone wants to set some custom headers based on `path and stats` with `X-` prefix.